### ISSUE TITLE
Setup a Makefile

### ISSUE THREAD

**Issue by fesch** [fesch/Structorizer.Desktop#587]
In https://github.com/fesch/Structorizer.Desktop/issues/537#issuecomment-418258560 @KimBrodowski requested to write a Makefile.

**Comment by GitMensch** [comment]
I request that, too.

Nice side effect: You can also automatically handle the Apple-specific stuff there.

**Comment by codemanyak** [comment]
I'm just afraid that the Make file will have to be created manually and maintained in parallel to the existing project configurations. Automatically, Eclipse only exports ant configurations (build.xml) or may be set up to work with Maven.
Well, okay, the Make configuration is not supposed to vary as frequently as the code but it is still going to be an additional peace of maintenance efforts. Just saying.

**Comment by codemanyak** [comment]
Just wondering: are the efforts to get `makeStructorizer` and the other existing `make...` files working (see e.g. #602) meant to be what @KimBrodowski was requesting?

**Comment by codemanyak** [comment]
@GitMensch 
Are the makefile efforts ready to be merged into bugfix branch to prepare the next version?

**Comment by codemanyak** [comment]
@KimBrodowski @GitMensch 
Can this issue be closed, i.e. have the spent efforts to update the make.... scripts met the request?

**Comment by codemanyak** [comment]
Seems that nobody will object if I close this issue now...
